shared/src/commonMain/kotlin/vn/luongvo/kmm/survey/domain/usecase/LogInUseCase.kt

```diff
@@ -0,0 +1,15 @@
+package vn.luongvo.kmm.survey.domain.usecase
+
+import kotlinx.coroutines.flow.Flow
+import vn.luongvo.kmm.survey.domain.repository.AuthRepository
+
+interface LogInUseCase {
+    operator fun invoke(email: String, password: String): Flow<String>
+}
+
+class LogInUseCaseImpl(private val repository: AuthRepository) : LogInUseCase {
+
```

Comment:
This seems to be the only place with extra line break, is the Kotlin convention acceptable for both 0 and 1 line break?

Reply:
@blyscuit The Android team prefers to have 1 line-break on top of the class and interface. I added the missing line breaks into:
- interface AuthRemoteDataSource
- interface AuthRepository

Reply:
Fixed in https://github.com/luongvo/kmm-survey/pull/45/commits/2b9bd7602a307a38f54093cce60ba0490f63deb9